README.md

```diff
@@ -436,6 +436,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [Neupy](http://neupy.com/pages/home.html) - Running and testing different Artificial Neural Networks algorithms.
 * [TensorFlow](https://github.com/tensorflow/tensorflow) - The most popular Deep Learning framework created by Google.
 * [Theano](https://github.com/Theano/Theano) - A library for fast numerical computation.
+* [Pytorch](http://pytorch.org/) - Tensors and Dynamic neural networks in Python with strong GPU acceleration.
```

Comment:
Would you mind sorting it in the alphabetical order?

Reply:
Thank you guys! Willie Parish